tools/ot/src/main/java/dk/alexandra/fresco/tools/cointossing/CoinTossing.java

```diff
@@ -0,0 +1,125 @@
+package dk.alexandra.fresco.tools.cointossing;
+
+import dk.alexandra.fresco.commitment.HashBasedCommitment;
+import dk.alexandra.fresco.commitment.HashBasedCommitmentSerializer;
+import dk.alexandra.fresco.framework.network.Network;
+import dk.alexandra.fresco.framework.network.serializers.ByteSerializer;
+import dk.alexandra.fresco.framework.util.AesCtrDrbg;
+import dk.alexandra.fresco.framework.util.ByteArrayHelper;
+import dk.alexandra.fresco.framework.util.Drbg;
+import dk.alexandra.fresco.framework.util.StrictBitVector;
+
+/**
+ * Class implementing two-party coin-tossing. That is, agreement on a random
+ * string of bits between two parties even if one of them is acting maliciously.
```

Comment:
You seem to have your formatter misconfigured. Linewrapping should be at 100 characters, not 80.

Reply:
I am trying to fix it when I pass over it. My checkstyle is weird. I have set it several times and it formats correctly for code, but comments it insists on doing a margin of 100 instead of 8.